ecep_cms/src/taccsite_cms/settings_custom.py

```diff
@@ -2,29 +2,30 @@
 # TACC WMA CMS SITE:
 # *.ECEP.TACC.UTEXAS.EDU
 
-# FAQ: Some _VARIABLES are duplicated from settings.py (but prefixed with "_")
-#      because current infrastructure lacks ability to reference default values
-
 ########################
-# DJANGO CMS SETTINGS
+# DJANGO_CMS
 ########################
 
 CMS_TEMPLATES = (
     ('standard.html', 'Standard'),
     ('fullwidth.html', 'Full Width'),
-
-    ('guide.html', 'Guide'),
-    ('guides/getting_started.html', 'Guide: Getting Started'),
-    ('guides/data_transfer.html', 'Guide: Data Transfer'),
-    ('guides/data_transfer.globus.html', 'Guide: Globus Data Transfer'),
-    ('guides/portal_technology.html', 'Guide: Portal Technology Stack')
```

Comment:
**Why?** Most are deprecated. The rest are default.

_**Why not just use default i.e. delete `CMS_TEMPLATES` here?** Cuz ECEP site has skilled (but not expert) editors, that maybe could receive Page Template privilege but would be confused by options they should not use._